src/Features/Core/Portable/Wrapping/AbstractWrapper.cs

```diff
@@ -47,7 +47,7 @@ protected AbstractSyntaxWrapper(IIndentationService indentationService)
             var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
             foreach (var item in nodesAndTokens)
             {
-                if (item == null || item.Span.IsEmpty)
+                if (item == null || item.Span.IsEmpty || item.IsMissing)
```

Comment:
technically missing tokens can have non-empty spans (fi the compiler attached things like whitespace or other trivia to them).  Being more resilient to that as well.